tests/test_many_scores/settings.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<settings>
+
+  <eigenvalue>
+    <batches>10</batches>
+    <inactive>5</inactive>
+    <particles>100</particles>
```

Comment:
Sometimes low particle counts can cause problems on tests using both OpenMP and MPI, i.e. too many fission sites banked for a single thread. Can we raise the number of particles a bit, maybe 500? and then just reduce the number of batches.